nasa-gibs/worldview
# WV-2953: Remove MODIS_Terra_Cholophyll_A and MODIS_Aqua_Chlorophyll_A

## Description

Fixes #WV-2953

Remove old MODIS_Terra_Cholophyll_A and MODIS_Aqua_Chlorophyll_A.


## How To Test

1. git checkout wv-2953-remove-old-chlorophyll-layers
2. npm run build && npm start
3. Go to Add Layers and type in "Chlorophyll"
4. Verify expected result - there should only be 3 chlorophyll a layers
